scripts/tests/vars.env

```diff
@@ -48,13 +56,7 @@ SECONDS_PER_SLOT=3
 SECONDS_PER_ETH1_BLOCK=1
 
 # Proposer score boost percentage
-PROPOSER_SCORE_BOOST=40
+PROPOSER_SCORE_BOOST=70
```

Comment:
Why this change? it's 40 in the spec: https://github.com/ethereum/consensus-specs/blob/dev/configs/mainnet.yaml#L88